plugin/auth_ldap/tests/mtr/auth_ldap_sasl.result

```diff
@@ -0,0 +1,44 @@
+INSTALL PLUGIN authentication_ldap_sasl SONAME 'authentication_ldap_sasl.so';
```

Comment:
This plugin is not being built by this change from what I see. So either this needs to be renamed together with variables and other stuff or some other change needs to be done.